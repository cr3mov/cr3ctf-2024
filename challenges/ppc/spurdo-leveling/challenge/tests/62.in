52137572880271351 949491241382858793
86 36078195301832
84 42589507578300
