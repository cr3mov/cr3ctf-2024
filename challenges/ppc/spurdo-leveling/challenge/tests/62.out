483418 5143 16714
